pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -211,7 +211,7 @@ public Object handle(Request request, Response response) {
                 String shortId = maybeParticipantByParticipantId.get().getProfile().map(Profile::getHruid).get();
                 addCollaboratorSampleId(tissueId, ddpInstance, ddpParticipantId, shortId);
             }
-            new TissueSMIDDao().createNewSMIDForTissueWithValue(tissueId, ffpeUser, smIdType, kitLabel);
```

Comment:
These two methods were doing the same thing so I removed `reateNewSMIDForTissueWithValue` and used `createNewSMIDForTissue`